dnf-behave-tests/features/repoquery/main.feature

```diff
@@ -0,0 +1,521 @@
+      top-a-1:2.0-1.src
+      top-a-1:2.0-1.x86_64
+      top-a-2:2.0-2.src
+      top-a-2:2.0-2.x86_64
+      """
+
+Scenario: dnf repoquery --all NAME
+ When I execute dnf with args "repoquery --all top-a"
+ Then the exit code is 0
+  And stdout is
+      """
+      top-a-1:1.0-1.src
+      top-a-1:1.0-1.x86_64
+      top-a-1:2.0-1.src
+      top-a-1:2.0-1.x86_64
+      top-a-2:2.0-2.src
+      top-a-2:2.0-2.x86_64
+      """
+
+
+# --available is the default, scenarios above should cover it
+Scenario: dnf repoquery --available NAME
+ When I execute dnf with args "repoquery --available top-a-2.0"
+ Then the exit code is 0
+  And stdout is
+      """
+      top-a-1:2.0-1.src
+      top-a-1:2.0-1.x86_64
+      top-a-2:2.0-2.src
+      top-a-2:2.0-2.x86_64
+      """
+
+
+# --arch
+Scenario: repoquery --arch ARCH
+ When I execute dnf with args "repoquery --arch noarch"
+ Then the exit code is 0
+  And stdout is
+      """
+      bottom-a1-1:1.0-1.noarch
+      bottom-a1-1:2.0-1.noarch
+      """
+
+Scenario: repoquery --arch ARCH (nonexisting arch)
+ When I execute dnf with args "repoquery --arch yesarch"
+ Then the exit code is 0
+  And stdout is empty
+
+
+# --cacheonly
+Scenario: repoquery -C (without any cache)
+ When I execute dnf with args "repoquery --available -C"
+ Then the exit code is 1
+ Then stdout is empty
+ Then stderr is
+      """
+      Cache-only enabled but no cache for 'repoquery-main'
+      Error: Cache-only enabled but no cache for 'repoquery-main'
+      """
+
+Scenario: repoquery -Cq (without any cache)
+ When I execute dnf with args "repoquery --available -Cq"
+ Then the exit code is 1
+ Then stdout is empty
+ Then stderr is
+      """
+      Cache-only enabled but no cache for 'repoquery-main'
+      Error: Cache-only enabled but no cache for 'repoquery-main'
+      """
+
+Scenario: repoquery -C (with cache)
+Given I successfully execute dnf with args "makecache"
+ When I execute dnf with args "repoquery -C mid*"
+ Then the exit code is 0
+ Then stdout is
+      """
+      mid-a1-1:1.0-1.src
+      mid-a1-1:1.0-1.x86_64
+      mid-a2-1:1.0-1.src
+      mid-a2-1:1.0-1.x86_64
+      """
+
+Scenario: repoquery -C (with cache, installed package)
+Given I successfully execute dnf with args "makecache"
+Given I successfully execute dnf with args "install bottom-a1"
+ Then Transaction is following
+      | Action        | Package                                   |
+      | install       | bottom-a1-1:2.0-1.noarch                  |
+ When I execute dnf with args "repoquery --installed -C"
+ Then the exit code is 0
+  And stdout is
+      """
+      bottom-a1-1:2.0-1.noarch
+      """
+
+Scenario: repoquery -C (with cache, but disabled repository)
+Given I successfully execute dnf with args "makecache"
+Given I disable the repository "repoquery-main"
+ When I execute dnf with args "repoquery --available -C"
+ Then the exit code is 0
+  And stdout is empty
+
+
+# --deplist
+Scenario: repoquery --deplist NAME
```

Comment:
There is a deplist feature file. What about to move it there. Also deplist is a command as an alias repoquery --deplist. Is there any test for it?

Reply:
Also valid for scenarios bellow.

Reply:
There's a rest for the alias in `features/repoquery/main.feature` and no other tests in other files, I think it's fine.